src/org/exist/util/VirtualTempFile.java

```diff
@@ -43,6 +43,7 @@
  * @author jmfernandez
  *
  */
+@Deprecated
 public class VirtualTempFile
```

Comment:
Marked deprecated .... It is used on quite some locations, please could you add a reference to its replacement? (or just mention it here)